ddl/ddl_api.go

```diff
@@ -1830,39 +1806,36 @@ func isIgnorableSpec(tp ast.AlterTableType) bool {
 // and returns the last charset and collate in options. If there is no charset in the options,
 // the returns charset will be "", the same as collate.
 func getCharsetAndCollateInTableOption(startIdx int, options []*ast.TableOption) (ca, co string, err error) {
```

Comment:
ca, co is not obviously clear.

Reply:
Sure...`charset` has been used by the `charset` package, so I changed `ca`, `co` to `chs` and `coll` respectively.